packages/shared/src/components/marketingCta/MarketingCtaPopoverSmall.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { ReactElement, useCallback, useEffect, useRef } from 'react';
+import classNames from 'classnames';
+import { Card } from '../cards/Card';
+import { CardCover } from '../cards/common/CardCover';
+import { CTAButton, Description, MarketingCta, Title } from './common';
+import { ViewSize, useBoot, useViewSize } from '../../hooks';
+import { useLogContext } from '../../contexts/LogContext';
+import { LogEvent, TargetType } from '../../lib/log';
+import { ButtonSize, ButtonVariant } from '../buttons/common';
+import { MiniCloseIcon } from '../icons';
+import { Button } from '../buttons/Button';
+
+export function MarketingCtaPopoverSmall({
+  marketingCta,
+}: {
+  marketingCta: MarketingCta;
+}): ReactElement {
+  const { title, description, image, ctaUrl, ctaText } = marketingCta.flags;
+  const { clearMarketingCta } = useBoot();
+  const { logEvent } = useLogContext();
+  const isImpressionTracked = useRef(false);
+  const isMobile = useViewSize(ViewSize.MobileL);
+
+  useEffect(() => {
+    if (isImpressionTracked.current) {
+      return;
+    }
+
+    logEvent({
+      event_name: LogEvent.Impression,
+      target_type: TargetType.MarketingCtaPopoverSmall,
+      target_id: marketingCta.campaignId,
+    });
+    isImpressionTracked.current = true;
+  }, [marketingCta.campaignId, logEvent]);
+
+  const onCtaClick = useCallback(() => {
+    logEvent({
+      event_name: LogEvent.Click,
+      target_type: TargetType.MarketingCtaPopoverSmall,
+      target_id: marketingCta.campaignId,
+    });
+    clearMarketingCta(marketingCta.campaignId);
+  }, [clearMarketingCta, marketingCta.campaignId, logEvent]);
+
+  const onCtaDismiss = useCallback(() => {
+    logEvent({
+      event_name: LogEvent.MarketingCtaDismiss,
+      target_type: TargetType.MarketingCtaPopoverSmall,
+      target_id: marketingCta.campaignId,
+    });
+    clearMarketingCta(marketingCta.campaignId);
+  }, [clearMarketingCta, marketingCta.campaignId, logEvent]);
+
+  return (
+    <Card
+      className={classNames(
+        '!p-0',
+        isMobile
+          ? '!max-h-none !border-none !bg-transparent !shadow-none'
+          : 'max-w-64 border-2 !border-accent-onion-default',
+      )}
+    >
+      {image && (
+        <CardCover
+          imageProps={{
+            loading: 'lazy',
+            alt: 'Post Cover',
+            src: image,
+            className: classNames('w-full', isMobile ? 'h-48' : '!rounded-14'),
+          }}
+        />
+      )}
+      <div
+        className={classNames(
+          'flex flex-col gap-4 p-4 pb-6 text-center',
+          isMobile ? '!pb-0' : 'pb-6',
+        )}
+      >
+        <Title className={classNames('!m-0', isMobile && '!typo-large-title')}>
+          {title}
+        </Title>
+        {description && (
+          <Description className={classNames(isMobile && '!typo-markdown')}>
+            {description}
+          </Description>
```

Comment:
Maybe best to improve the components rather then adding more important rules, since these components are supposed to be common for marketingCta

Reply:
What would you suggest?

Reply:
Maybe something like we do with buttons, of course this would be a way way smaller component, but that could have a default typo, and then accept a `typo` prop from enum of all the allowed title typos for this component

Reply:
So CTA `Title` inherits `CardTitle` and which in turn inherits from `Title` :triggered:. So passing typo to that one would require a change to 2 other components... And `Description` is just a `classed('p', 'text-text-secondary typo-callout')`

The latter is easier to adjust, but I think the former can just be left alone?

Reply:
Yeah should be fine for now